### PR TITLE
Ddf 2592 backport

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Backports DDF-2592: Fixes issues of running a parallel quickbuild on Windows. and adds UTF-8 to integration tests.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@brendan-hofmann @ricklarsen 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Build](https://github.com/orgs/codice/teams/build)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@pklinef
@stustison
#### How should this be tested? (List steps with links to updated documentation)
Full build passes on windows and *nix.
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by shaundmorris** [comment]
@AzGoalie can you please update the summary to be more accurate?

File: distribution/docs/pom.xml [hunk 1]
```diff
@@ -57,41 +57,29 @@
             </activation>
                 <properties>
                     <asciidoctor.maven.plugin.version>1.5.3</asciidoctor.maven.plugin.version>
+                    <asciidoctorj.version>1.5.4.1</asciidoctorj.version>
+                    <asciidoctorj.diagram.version>1.5.0</asciidoctorj.diagram.version>
+                    <asciidoctorj.pdf.version>1.5.0-alpha.11</asciidoctorj.pdf.version>
+                    <jruby.version>1.7.25</jruby.version>
```
Comment (ricklarsen) [inline distribution/docs/pom.xml]: 👍

File: distribution/docs/pom.xml [hunk 2]
```diff
@@ -57,41 +57,29 @@
...
-                        <scope>provided</scope>
-                        <exclusions>
-                            <exclusion>
-                                <groupId>rubygems</groupId>
-                                <artifactId>asciidoctor</artifactId>
-                            </exclusion>
-                        </exclusions>
+                        <groupId>org.asciidoctor</groupId>
+                        <artifactId>asciidoctorj</artifactId>
+                        <version>${asciidoctorj.version}</version>
```
Comment (ricklarsen) [inline distribution/docs/pom.xml]: 👍

File: distribution/docs/pom.xml [hunk 3]
```diff
@@ -57,41 +57,29 @@
...
                         <groupId>org.asciidoctor</groupId>
-                        <artifactId>asciidoctorj</artifactId>
-                        <version>1.5.4</version>
+                        <artifactId>asciidoctorj-diagram</artifactId>
+                        <version>${asciidoctorj.diagram.version}</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.jruby</groupId>
+                        <artifactId>jruby-complete</artifactId>
+                        <version>${jruby.version}</version>
```
Comment (ricklarsen) [inline distribution/docs/pom.xml]: 👍

File: distribution/docs/pom.xml [hunk 4]
```diff
@@ -235,7 +220,7 @@
                                 </execution>
                                 <execution>
                                     <id>generate-pdf-doc</id>
-                                    <phase>process-resources</phase>
+                                    <phase>generate-resources</phase>
```
Comment (ricklarsen) [inline distribution/docs/pom.xml]: 👍

File: distribution/docs/pom.xml [hunk 5]
```diff
@@ -57,41 +57,29 @@
...
                     <asciidoctor.maven.plugin.version>1.5.3</asciidoctor.maven.plugin.version>
+                    <asciidoctorj.version>1.5.4.1</asciidoctorj.version>
+                    <asciidoctorj.diagram.version>1.5.0</asciidoctorj.diagram.version>
+                    <asciidoctorj.pdf.version>1.5.0-alpha.11</asciidoctorj.pdf.version>
+                    <jruby.version>1.7.25</jruby.version>
                     <packaging>pom</packaging>
                     <module>docs</module>
                     <skipTests>false</skipTests>
                 </properties>
-                <repositories>
```
Comment (pklinef) [inline distribution/docs/pom.xml]: Please make a ticket/PR to do this on master as well.

**Comment by brendan-hofmann** [comment]
Successfully heroed. 👍

**Comment by AzGoalie** [comment]
bamboo 'failed' but the build passed
`build	12-Dec-2016 15:26:09	[INFO] ------------------------------------------------------------------------
build	12-Dec-2016 15:26:09	[INFO] BUILD SUCCESS
build	12-Dec-2016 15:26:09	[INFO] ------------------------------------------------------------------------
build	12-Dec-2016 15:26:09	[INFO] Total time: 01:55 h
build	12-Dec-2016 15:26:09	[INFO] Finished at: 2016-12-12T22:26:09+00:00
build	12-Dec-2016 15:26:12	[INFO] Final Memory: 820M/2500M
build	12-Dec-2016 15:26:12	[INFO] ------------------------------------------------------------------------
simple	12-Dec-2016 15:26:12	Parsing test results under /home/bamboo/bamboo-agent-home/xml-data/build-dir/DDF-PRINC-JOB1...
error	12-Dec-2016 15:26:42	Failed to parse test result file "/home/bamboo/bamboo-agent-home/xml-data/build-dir/DDF-PRINC-JOB1/ddf/catalog/common/catalog-common-geo-formatter/target/surefire-reports/TEST-ddf.geo.formatter.TestPolygon.xml"
error	12-Dec-2016 15:27:02	Error occurred while running Task 'Build Changed DDF Modules and Dependents(4)' of type com.atlassian.bamboo.plugins.maven:task.builder.mvn3.
error	12-Dec-2016 15:27:02	java.lang.OutOfMemoryError: GC overhead limit exceeded`
        `

**Comment by emanns95** [comment]
Passed full build on MacOS :+1:

**Pull request merged**